src/_scss/pages/explorer/detail/header/header.scss

```diff
@@ -79,7 +79,7 @@
 
         .detail-header__icon {
           padding: 0 0 0 rem(5);
-          cursor: default;
+          cursor: pointer;
```

Comment:
this can remain as cursor: default; our other tooltips don't have pointer

Reply:
Done